tmt/steps/report/reportportal.py

```diff
@@ -14,77 +14,139 @@
 
 @dataclasses.dataclass
 class ReportReportPortalData(tmt.steps.report.ReportStepData):
+
+    @staticmethod
+    def _flag_env_to_default(option: str, default: bool) -> bool:
+        env_var = 'TMT_PLUGIN_REPORT_REPORTPORTAL_' + option.upper()
+        if env_var not in os.environ:
+            return default
+        return bool(os.getenv(env_var) == '1')
+
+    @overload
+    @staticmethod
```

Comment:
I guess the combination of `staticmethod` and `overload` is confusing package build on RHEL9, ruining the tests. I'm afraid these methods would need to become functions, i.e. be moved outside of the `ReportReportPortalData` class.